frontend/src/pages/start/libs/components/voice-tone-form/libs/constants.ts

```diff
@@ -0,0 +1,20 @@
+import { type SelectOption } from '~/libs/types/types.js';
+import {
+  type SiteCreateStepTone,
+  SiteToneType,
+} from '~/packages/sites/sites.js';
+
+import { SiteToneTypeToReadable } from './site-tone-type-to-readable.enum.js';
+
+const OPTIONS: SelectOption<string | number>[] = Object.values(
```

Comment:
Looks like it's not `string | number` - you actually know the type of values passed (`SiteToneType` vals)